src/template.html

```diff
@@ -14,11 +14,11 @@ <h1 class="text-5xl mt-2 mx-2 text-white font-extrabold">Leaderboard</h1>
         <div class="w-full">
           <div class="flex fex-row justify-between px-2 md:px-10">
             <h2 class="text-2xl font-bold">Recent Score</h2>
-            <button type="button" class="py-1 px-2 border bg-slate-400 font-bold refresh">Refresh</button>
+            <button type="button" class="refresh border-blue-600 rounded-md bg-blue-800 text-white border px-3 py-1 font-bold self-end ">Refresh</button>
           </div>
-          <table class="table-auto border-2 border-black w-full mt-10">
-            <thead class="border-4 border-black">
-              <tr>
+          <table class="table-auto border-2 border-blue-600 w-full mt-10">
+            <thead class="border-4 border-blue-600">
+              <tr class="bg-blue-600 text-white">
```

Comment:
You did really well well your project, please kindly change the placeholder on the second input box to `your score` as shown in the design given as it is good to follow the given framework for your design 😄 

Your design :
![leadership board scst](https://github.com/AsumVictor/Leadership-board/assets/20537712/dec3f7c0-7374-499a-b7bf-107871b39710)

Framework given :
![image](https://github.com/AsumVictor/Leadership-board/assets/20537712/2b429263-3890-46ec-a328-73965f270026)

Reply:
Thank you very much! I have added the appropriate placeholder for the score input